_config.yml

```diff
@@ -13,8 +13,8 @@
 # you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
 # You can create any custom variable you would like, and they will be accessible
 # in the templates via {{ site.myvariable }}.
-title: 'Open Disclosure California'
-email: caciviclab@googlegroups.com
+title: 'Open Disclosure Oakland'
+email: oaklandopendisclosure@gmail.com
```

Comment:
Is this a new email address you just made? If so, I think we may instead consider using a group via OpenOakland's G Suite? E.g. `opendisclosure@openoakland.org`